breacharbiter.go

```diff
@@ -103,11 +143,9 @@ func (b *breachArbiter) Start() error {
 
 	// For each of the channels read from disk, we'll create a channel
 	// state machine in order to watch for any potential channel closures.
-	channelsToWatch := make([]*lnwallet.LightningChannel,
-		len(activeChannels))
+	channelsToWatch := make([]*lnwallet.LightningChannel, len(activeChannels))
 	for i, chanState := range activeChannels {
-		channel, err := lnwallet.NewLightningChannel(nil, b.notifier,
-			chanState)
+		channel, err := lnwallet.NewLightningChannel(nil, b.notifier, chanState)
```

Comment:
Same here, the prior formatting was correct as the line exceeded 80 characters. It's recommended the formatting modifications like this be kept out of PR's as it generates a smaller diff which makes things easier on the reviewer. On a similar note extraneous code movement within a PR also adds to this effect.